src/components/Chat/customStyles.js

```diff
@@ -0,0 +1,14 @@
+export const customStyles = {
```

Comment:
- As I can see there are only CSS, why can you put them in you SCSS file?
- If can get it well, this is to style the popup chat modal, `the modal` is always `60%` of the screen, cant it be somehow bad for the small screen. the remaining 40% is a huge space

Reply:
Changes applied